src/style.css

```diff
@@ -47,12 +49,26 @@ body {
   /* outline: red solid 2px; */
 }
 
-#parent .input {
-  width: calc(100% - 24px);
-  padding: 12px;
+.div_input {
+  border-bottom: gray solid 1px;
+}
+
+#parent .input,
+#parent .desc {
+  width: calc(100% - 60px);
+  padding: 20px 12px;
   font-style: italic;
+  font-size: 15px;
   border: 0;
-  border-bottom: gray solid 1px;
+}
+
+#parent .desc {
+  width: calc(100% - 40px);
+  padding: 12px;
+  margin: 0;
+  border: 0;
+
+  /* outline: red solid 2px; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: